src/index.ts

```diff
@@ -357,6 +356,22 @@ async function wranglerCommands() {
 					setOutput("deployment-alias-url", aliasUrl);
 				}
 			}
+			// Check if this command is a pages deployment
+			if (
+				command.startsWith("pages publish") ||
+				command.startsWith("pages deploy")
+			) {
+				const pagesArtifactFields =
+					await getDetailedPagesDeployOutput(wranglerOutputDir);
+
+				if (pagesArtifactFields) {
```

Comment:
Maybe if `pagesArtifactFields` is null, we can log a message that outputs haven't been set, maybe due to being on an old wrangler version?

Reply:
Hmm, I was thinking an info log like we do in a couple places already https://github.com/cloudflare/wrangler-action/blob/main/src/index.ts#L118

What do you mean by logging in the deployment details?